### ISSUE TITLE
Incorrect schema in Data Dictionary.txt

### ISSUE THREAD

**Issue by liorbp** [NationalGalleryOfArt/opendata#12]
The schema for **objects** table in Data Dictionary.txt indicates that uuid is included in the table, but it's not.
Should be updated.

**Comment by beaudet** [comment]
Yep, it does seem to be missing.  That's an unintentional omission and we'll get it fixed soon.  Sorry for the inconvenience.

**Comment by Ennego** [comment]
Hi @andy-jdl -- Thanks for the question and interest. If you're looking for images of art, you can use the IIIF endpoint for images listed in the [published images table](https://github.com/NationalGalleryOfArt/opendata/blob/main/data/published_images.csv), which are tied the objects table via the "depictstmsobjectid" column.
